Notebook failure since new year

https://github.com/Ouranosinc/PAVICS-landing/blob/master/content/notebooks/hydrology/PAVICStutorial_Hydrology-01_Intro.ipynb

```
20:17:31  =================================== FAILURES ===================================
20:17:31  _ PAVICS-landing-master/content/notebooks/hydrology/PAVICStutorial_Hydrology-01_Intro.ipynb::Cell 3 _
20:17:31  Notebook cell execution failed
20:17:31  Cell 3: Cell outputs differ
20:17:31  
20:17:31  Input:
20:17:31  print(gr4jcn.rvi)
20:17:31  
20:17:31  Traceback:
20:17:31   mismatch 'stdout'
20:17:31  
20:17:31   assert reference_output == test_output failed:
20:17:31  
20:17:31    ':Calendar   ...id GR4JCN\n\n' == ':Calendar   ...id GR4JCN\n\n'
20:17:31    
20:17:31    Skipping 1808 identical leading characters in diff, use -v to show
20:17:31    - o.ca/ (2026).
20:17:31    ?           ^
20:17:31    + o.ca/ (2025).
20:17:31    ?           ^
20:17:31      :NetCDFAttribute model_id GR4JCN
```

--- a/notebooks/output-sanitize.cfg
+++ b/notebooks/output-sanitize.cfg
@@ -30,6 +30,10 @@ replace: [DATE_TIME]
 regex: [\/| ]\d{4}-\d{2}-\d{2}[\/| ]
 replace: /DATE/
 
+[copyright_year]
+regex: \(\d{4}\)
+replace: (COPYRIGHT_YEAR)
+
 [time_stamp]
 regex: [\/| ]\d{2}:\d{2}:\d{2}[\/| ]
 replace: /TIME/

